packages/react/tsup.config.ts

```diff
@@ -0,0 +1,8 @@
+export default {
+  sourcemap: true,
```

Comment:
Can you add here dts: true and remove `--dts` on the "build" command

Reply:
Should I do this for `packages/sdk` as well?